.eslintrc

```diff
@@ -14,14 +14,9 @@
     },
 
     "globals": {
-        "__baseDir": false,
-        "logger" : false
+        "__baseDir": false
```

Comment:
Clara 0.4.x renamed this to ``mainDir.``

Reply:
It hasn't been renamed here.